pkg/apis/experimental/install/install.go

```diff
@@ -51,14 +51,16 @@ func init() {
 		GroupVersion: groupVersion,
```

Comment:
Oh-- this is from a list. I see. Sorry!

On Mon, Sep 21, 2015 at 3:29 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/apis/experimental/install/install.go
> <https://github.com/kubernetes/kubernetes/pull/14156#discussion_r40034863>
> :
>
> > @@ -51,14 +51,16 @@ func init() {
> >  		GroupVersion: groupVersion,
>
> because we ask people to specify the versions in the order of the most
> preferred first,
>
>
> https://github.com/kubernetes/kubernetes/blob/master/pkg/api/registered/registered.go#L42
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14156/files#r40034863>.
>